packages/front-end/components/Experiment/DateResults.tsx

```diff
@@ -177,10 +182,10 @@ const DateResults: FC<{
                   // For non-baseline variations and cumulative turned ON, calculate uplift from cumulative data
                   else if (i) {
                     const crA = totalUsers[0]
-                      ? totalValue[0] / totalUsers[0]
+                      ? totalValue[0] / totalDenominator[0]
```

Comment:
might need a divide-by-zero check

Reply:
oh shoot, thanks, the ifelse logic needs to use totalDenominator